src/models/pang13/README.md

```diff
@@ -0,0 +1,56 @@
+# Pangolin (pang13)
+
+- [External Overview](./external-overview.md)
+- [Internal Overview](./internal-overview.md)
+- [Parts & Repairs](./repairs.md)
+
+![Pangolin](./img/pang13.webp)
+
+The System76 Pangolin is a laptop with the following specifications:
+
+- CPU
+    - Supports AMD Ryzen (Rembrandt) Mobile CPUs
+        - [Ryzen 7 7735U](https://www.amd.com/en/products/apu/amd-ryzen-7-7735u)
+- BIOS
+    - Winbond W25R256JW flash chip
+        - WSON-8 form factor
+    - Programmed with System76 firmware (non-open)
+- EC
+    - ITE IT5571
+        - Programmed with non-open EC firmware
+- Graphics
+    - GPU: AMD Radeon 680M
+    - eDP display: 15.6" 1920x1080@144Hz LCD
+        - LCD panel: Innolux N156HRA-EA1 (or equivalent)
+    - External video output: 
+        - 1x HDMI 2.0
+        - 1x DisplayPort 1.4 over USB-C
+- Memory
+    - 32GB LPDDR5 (on-board) @ 6400 MHz
+        - 16x [Micron MT62F2G32D8DR-031](https://www.micron.com/products/dram/lpdram/part-catalog/mt62f2g32d8dr-031-wt) (2GB each)
+- Networking
+    - Gigabit Ethernet
+    - M.2 PCIe/CNVi WiFi/Bluetooth
+        - Mediatek MT7921K
+- Power
+    - 65W (19V, 3.42A) DC-in port
+        - Barrel size: 5.5mm (outer), 2.5mm (inner)
+    - Included AC adapter: Shenzhen Jihongda JHD-AD065B-190342BA-A
+        - AC power cord type: IEC C5
+    - USB-C charging compatible with 65W+ charger
```

Comment:
Should be 90W+ charger

Reply:
Updated with 416fcd506c1c8f2fbe82797d790557557b4a77e2.